### PR TITLE
[new release] simple63 (1)

### PR DESCRIPTION
**Pull request by barko**
Integer compression and decompression module

- Project page: <a href="https://github.com/barko/simple63">https://github.com/barko/simple63</a>
- Documentation: <a href="https://barko.github.io/simple63/">https://barko.github.io/simple63/</a>

##### CHANGES:

* first `opam` release

### CODE REVIEW

File: packages/simple63/simple63.1/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   non-negative integer sequences. It is similar to the Simple-8b
+   technique, adapted to work with OCaml's 63-bit integers."
+maintainer: ["mika@illouz.net"]
+authors: ["Mika Illouz"]
+license: "BSD"
+homepage: "https://github.com/barko/simple63"
+doc: "https://barko.github.io/simple63/"
+bug-reports: "https://github.com/barko/simple63/issues"
+depends: [
+  "dune" {> "1.5"}
```
Comment (mseri) [inline packages/simple63/simple63.1/opam:14]: Your dune-project file is requiring 2.6
```suggestion
  "dune" {> "2.6"}
```

File: packages/simple63/simple63.1/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "ocaml" {>= "4.08.0"}
+  "ppx_inline_test"
```
Comment (mseri) [inline packages/simple63/simple63.1/opam:16]: This is required even if you don't run the tests?

File: packages/simple63/simple63.1/opam (hunk 1 above)
Comment (mseri) [reply packages/simple63/simple63.1/opam]: Can you make this change also upstream?

File: packages/simple63/simple63.1/opam (hunk 2 above)
Comment (barko) [reply packages/simple63/simple63.1/opam:16]: No. I'll add {with-test}, change dune to {> "2.6"} and resubmit. Thx for reviewing.

File: packages/simple63/simple63.1/opam [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@doc" {with-doc}
```
Comment (kit-ty-kate) [inline packages/simple63/simple63.1/opam:28]:
```suggestion
    "@install"
    "@doc" {with-doc}
```
The previous question from @mseri still remains though. If no test is ran, `ppx_inline_test` is not required. We prefer to run the tests when possible if they are not unstable. Could you add `"@runtest" {with-test}` between those two lines to launch the tests if that's the case?

Comment (barko) [reply packages/simple63/simple63.1/opam]: Ok! Trying again and thanks for your patience.

**Comment by mseri** [comment]
Looks like ppx_inline_test has to be htere evn when no test is run: 

```
#=== ERROR while compiling simple63.1 =========================================#
# context              2.0.7 | linux/x86_64 | ocaml-base-compiler.4.11.1 | pinned(https://github.com/barko/simple63/releases/download/v1/simple63-v1.tbz)
# path                 ~/.opam/4.11/.opam-switch/build/simple63.1
# command              ~/.opam/4.11/bin/dune build -p simple63 -j 72 @install
# exit-code            1
# env-file             ~/.opam/log/simple63-23-4aef26.env
# output-file          ~/.opam/log/simple63-23-4aef26.out
### output ###
# File "dune", line 5, characters 19-34:
# 5 |   (preprocess (pps ppx_inline_test))
#                        ^^^^^^^^^^^^^^^
# Error: Library "ppx_inline_test" not found.
# Hint: try:
#   dune external-lib-deps --missing -p simple63 -j 72 @install
```

**Comment by mseri** [comment]
I am not allowed to edit the file. I think we just need to remove {with-test} from the ppx dependency and then it is ready to go. Sorry for the double work

**Comment by kit-ty-kate** [comment]
> EDIT: please @barko do nothing for now. @kit-ty-kate do you see a reason for it to compile fine on Travis but fail on the CI with the error above?

The installation was simply ignored because of https://github.com/ocaml/opam-repository/issues/17152. But it's fixed now so it should be consistent next time this PR is updated.

**Comment by barko** [comment]
So ... should I remove `{with-test}` on `"ppx_inline_test"` and resubmit?

**Comment by kit-ty-kate** [comment]
yes, sorry for the bother.

If that's ok with you, next time you can allow us to modify the branch (there is a checkbox when creating the PR), it might be less of a bother for you.

**Comment by barko** [comment]
re checkbox: I'm using `dune-release` to do the submission, so there's no box to check.

I resubmitted, and `dune-release` decided to create a new PR #17188 rather than amend this one. Not sure why.

**Comment by kit-ty-kate** [comment]
Not sure either. cc @NathanReb 

Closing then.

**Pull request closed without merging**